File: java/Konstruktor/src/main/java/org/example/Clovek.java
package org.example;

public class Clovek {
    //atributy (promenne nalezici tride)
    private String jmeno;
    private int vek;
    private Boolean jeDospely;

    public String getJmeno()
    {
        return jmeno;
    }
    public int getVek()
    {
        return vek;
    }
    private void setJmeno(String jmeno)
    {
        this.jmeno = jmeno;
    }
    public void setVek(int vek)
    {
        this.vek = vek;
        this.jeDospely = vek >= 18;
    }

    //konstruktor
    public Clovek(String jmeno, int vek)
    {
        this.jmeno = jmeno;
        this.vek = vek;
        this.jeDospely = vek >= 18;
    }

    //metody (funkce nalezici tride)
    public void pozdrav()
    {
        System.out.println("Ahoj posila " + jmeno + " ve veku " + vek + " je dospely " + jeDospely);
    }
}

File: csharp/Vlastnosti/Vlastnosti/Clovek.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Vlastnosti
{
    internal class Clovek
    {

        //vlastnosti
        public string Jmeno { get; private set; }

        //private string jmeno;
        //public string GetJmeno()
        //{
        //    return jmeno;
        //}
        //protected void SetJmeno(string jmeno)
        //{
        //    this.jmeno = jmeno;
        //}



        private int vek;
        public int Vek {
            get { return vek; }
            set {
                vek = value;
                jeDospely = value >= 18; }
        }

        //private int vek;
        //public int GetVek()
        //{
        //    return vek;
        //}

        //public void SetVek(int vek)
        //{
        //    this.vek = vek;
        //    this.jeDospely = vek >= 18;
        //}


        //atributy (promenne nalezici tride)
        private bool jeDospely;

        //konstruktor
        public Clovek(string jmeno, int vek)
        {
            this.Jmeno = jmeno;
            this.Vek = vek;
            //this.jeDospely = vek >= 18;
        }

        //metody (funkce nalezici tride)
        public void Pozdrav()
        {
            Console.WriteLine($"Ahoj posila {Jmeno} ve veku {Vek} je dospely {jeDospely}");
        }
    }
}
